packages/babel-make-styles/README.md

```diff
@@ -18,6 +18,25 @@ yarn add @fluentui/babel-make-styles
 }
 ```
 
+### Import `makeStyles()` from custom packages
+
+```js
+import { makeStyles } from 'custom-package';
```

Comment:
just curious, is this already handled ? 

```
import { makeStyles as customMakeStyles } from 'custom-package';
```

Reply:
As we are using `.referencesImport()`, it's handled 😉

Reply:
Added `import-alias` fixture to be sure 👌